api/client/kill.go

```diff
@@ -17,17 +18,16 @@ func (cli *DockerCli) CmdKill(args ...string) error {
 
 	cmd.ParseFlags(args, true)
 
-	var errNames []string
+	var errs []string
 	for _, name := range cmd.Args() {
 		if err := cli.client.ContainerKill(name, *signal); err != nil {
-			fmt.Fprintf(cli.err, "%s\n", err)
```

Comment:
@duglin The problem here is we have to return `error` in `CmdKill`, if we `fmt.Fprintf` here, what do you expect for the return value? I don't think it's a good idea to change return value types here.
And I don't think we have an order issue, slices will keep the order as we appended, no?

Yeah I'm being optimistic since I don't see much options here, but I'm always willing and ready for comments and modifications. :smile: